### PR TITLE
Update based on comments of EthanG and DavidB

### PR DESCRIPTION
**Pull request by vb-balazs**
Update based on the comments of Ethan Grossman (01.10) and David Black (04.10).

### CODE REVIEW

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml [hunk 1]
```diff
@@ -240,55 +242,55 @@
...
-	There are two major characteristics to the data plane:
+	There are two major characteristics to the data plane: the technology and 
+	the encapsulation, as discussed below.
         </t>
+		<section title="Data Plane Technology" anchor="sec_dp_tech">
          <t>
-         <list style='numbers'>
-         <t>
-         Data plane technology: The DetNet data plane is provided by the DetNet 
-		 service and forwarding sub layers.  The DetNet service sub-layer
```
Comment (louberger) [inline data-plane-framework/draft-ietf-detnet-data-plane-framework.xml:260]: I think the sentence "The DetNet data plane is provided by the DetNet service and forwarding sub layers. "  Should not be dropped, suggest adding back as the frst sentence in the new section.

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml [hunk 2]
```diff
@@ -240,55 +242,55 @@
...
          </t>
+		</section>
+		<section title="Data Plane Format" anchor="sec_dp_form">
          <t>
-          Encapsulation format: DetNet encodes specific flow attributes
-	  (namely flow identity and sequence number) in packets. 
+          DetNet encodes specific flow attributes
+	  (flow identity and sequence number) in packets. 
 	  For example, in DetNet IP, 
           zero encapsulation may be used and no sequence number
```
Comment (louberger) [inline data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: suggest, s/may be/is

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml [hunk 3]
```diff
@@ -240,55 +242,55 @@
...
+     </section>
 
-  +-------+  +---------+
-  | DN IP |  | DN MPLS |
-  +-------+  +---------+
+      <section title="Encapsulation" anchor="sec_dp_encap">
+         <t>
+		 The encapsulation of a DetNet flow allows it to be sent over a   
+		 data plane technology other than its native type. For example, 
+		 an Ethernet TSN app flow can be sent as a DetNet app flow over MPLS.
```
Comment (louberger) [inline data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: This sentence isn't quit right for IP.  How about:

DetNet uses header information to perform traffic classification, i.e., identify DetNet flows, and provide DetNet service and forwarding functions.  As mentioned above, DetNet may add headers, as is the case for DN MPLS, or may headers that are already present, as is the case in DN IP.

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml [hunk 4]
```diff
@@ -340,16 +333,18 @@
...
-            sub-layer specific functions (e.g., PREOF). DetNet forwarding
-            sub-layer related functions require only Flow-ID. 
-         </t>
-         <t>
-           Metadata can be a useful way of identifying packets that need to be
-           treated as a flow or flow aggregate. It is also useful as a way of
-           including a sequence number the packet for use by the PREOF function
-           or as a place to carry OAM indications or OAM information to
-           instrument DetNet data plane operation.
+		  The DetNet data plane supports a Flow-ID (for identification of the
```
Comment (louberger) [inline data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: s/data plane/data plane framework

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml [hunk 5]
```diff
@@ -442,24 +437,27 @@
...
+  </t>
+  <section title="Reservation and Allocation of resources">
+      <t>
         Reservation of resources can allocate resources to specific DetNet
-        flows.  This can eliminate packet contention and loss for DetNet
-        traffic. This also can reduce jitter for the DetNet traffic.  DetNet
-        flows are assumed to behave with respect to the reserved traffic
-        profile. If other traffic shares the link resources, the use of
+        flows.  This can eliminate packet contention and packet loss for DetNet
+        traffic. This also can reduce jitter for DetNet traffic.  So, resources
```
Comment (louberger) [inline data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: s/So, r/R

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml [hunk 6]
```diff
@@ -885,11 +893,10 @@ Sub-Network        |  L2  |  | TSN  |  | UDP  |
...
       <t>
-        Flow aggregation includes aggregation accomplished through the use of
-        hierarchical LSPs in MPLS and tunnels, in the case of IP, MPLS and TSN,
-        all of which aggregate multiple DetNet flows into a single new DetNet
-        flow.  Aggregation can also be grouping of IP flows that share
-        6-tuple attributes or flow identifiers at the DetNet sub-layer.
+        Flow aggregation means that multiple App-flows are served by a single new DetNet
+        flow.  There are many techniques to achieve aggregation, for example in case of IP, 
+		it can be grouping of IP flows that share 6-tuple attributes or flow 
+		identifiers at the DetNet sub-layer.
```
Comment (louberger) [inline data-plane-framework/draft-ietf-detnet-data-plane-framework.xml:905]: Why drop the MPLS example that are very familiar to some. Suggest adding back:

Another example includes aggregation accomplished through the use of
hierarchical LSPs in MPLS and tunnels.

**Review by louberger — CHANGES_REQUESTED** [review]

**Comment by agmalis** [comment]
I agree with Lou, it's a well-known and deployed example. I'm not sure why
it was removed.

Cheers,
Andy

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml (hunk 1 above)
Comment (vb-balazs) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml:260]: This was already clarified earlier in the draft. Yes, it is valid, but a already said. That is the reason for the removal.

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml (hunk 2 above)
Comment (vb-balazs) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: OK. Change "may be --> is".

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml (hunk 3 above)
Comment (vb-balazs) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: OK. Agree with proposed change.

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml (hunk 4 above)
Comment (vb-balazs) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: OK.

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml (hunk 5 above)
Comment (vb-balazs) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: OK.

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml (hunk 6 above)
Comment (vb-balazs) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml:905]: OK.

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml (hunk 1 above)
Comment (louberger) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml:260]: In reading the section, I think the sentence still has value,

Comment (EthanGrossman) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml:260]: I was the one who suggested that it felt redundant in the context of reading the draft end-to-end.  I don't object to putting it back in.

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml (hunk 3 above)
Comment (EthanGrossman) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: Typo at "or may headers", I assume s/b "or may use headers".

**Review by louberger — APPROVED** [review]
comments have been resolved

**Pull request merged**